py/server/deephaven/plot/figure.py

```diff
@@ -104,9 +104,9 @@ class Figure(JObjectWrapper):
 
     j_object_type = jpy.get_type("io.deephaven.plot.Figure")
 
-    def __init__(self, j_figure: jpy.JType = None):
+    def __init__(self, rows: int = 1, cols: int = 1, j_figure: jpy.JType = None):
         if not j_figure:
-            self.j_figure = _JPlottingConvenience.figure()
+            self.j_figure = _JPlottingConvenience.figure(rows, cols)
```

Comment:
@mofojed figure.py is generated, see
```
######################################################################################################################
#               This code is auto generated. DO NOT EDIT FILE!
# Run generatePythonFigureWrapper or "./gradlew :Generators:generatePythonFigureWrapper" to generate
######################################################################################################################
```
We need to update 'io.deephaven.plot.PlotConvenience' first, and that may lead to other changes in the plot library which is a mixture of hand-written and generated code itself.

Reply:
Right - I totally forgot that file was generated. I'm an idiot, I've edited this before.
I'll see if I can make the proper changes in PlottingConvenience and such.